Lib/tracemalloc.py

```diff
@@ -282,6 +282,13 @@ def size(self):
     def traceback(self):
         return Traceback(self._trace[2])
 
+    @property
+    def length(self):
```

Comment:
I suggest to rename it total_nframe and move it to Traceback.

Reply:
Ok!